File: 프로그래머스/lv1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    answer = True
    tmp = 0
    arr=list(s)
    for i in arr:
        if i=="p" or i=="P":
            tmp += 1
        elif i=="y" or i=="Y":
            tmp -= 1
    
    if tmp!=0: answer=False
    
    return answer

File: 프로그래머스/lv1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        char[] arr = s.toLowerCase().toCharArray();
		
        int p = 0, y = 0;
		for(int i=0; i<arr.length; i++) {
			if(arr[i] == 'p') p++;
			else if(arr[i] == 'y') y++;
		}
        return p == y;
    }
}